src/View/Components/Card.php

```diff
@@ -22,28 +23,63 @@ class Card extends Component
 
     public ?string $footer;
 
-    public ?string $cardClasses;
+    public ?string $cardClasses = '';
+
+    public ?string $headerClasses = '';
+
+    public ?string $footerClasses = '';
 
     public function __construct(
         ?string $padding = 'px-2 py-5 md:px-4',
         ?string $shadow = 'shadow-md',
         ?string $rounded = 'rounded-lg',
-        ?string $color = 'bg-white border border-secondary-200 dark:bg-secondary-800 dark:border-0',
+        ?string $color = 'bg-white dark:bg-secondary-800',
         ?string $title = null,
         ?string $action = null,
         ?string $header = null,
         ?string $footer = null,
-        ?string $cardClasses = null
+        ?string $cardClasses = '',
+        ?string $headerClasses = '',
+        ?string $footerClasses = '',
     ) {
-        $this->padding     = $padding;
-        $this->shadow      = $shadow;
-        $this->rounded     = $rounded;
-        $this->color       = $color;
-        $this->title       = $title;
-        $this->action      = $action;
-        $this->header      = $header;
-        $this->footer      = $footer;
-        $this->cardClasses = $cardClasses;
+        $this->padding       = $padding;
+        $this->shadow        = $shadow;
+        $this->rounded       = $rounded;
+        $this->color         = $color;
+        $this->title         = $title;
+        $this->action        = $action;
+        $this->header        = $header;
+        $this->footer        = $footer;
+        $this->cardClasses   = $this->setCardClasses($cardClasses);
+        $this->headerClasses = $this->setHeaderClasses($headerClasses);
+        $this->footerClasses = $this->setFooterClasses($footerClasses);
+    }
+
+    public function setCardClasses($cardClasses)
+    {
+        return Str::of('w-full flex flex-col')->finish(' ')
+            ->append($this->shadow)->finish(' ')
+            ->append($this->rounded)->finish(' ')
+            ->append($this->color)->finish(' ')
+            ->append($cardClasses);
+    }
```

Comment:
Missing split lines and the function return type

```suggestion
    public function setCardClasses(?string $cardClasses): string
    {
        return Str::of('w-full flex flex-col')
            ->append(" {$this->shadow}")
            ->append(" {$this->rounded}")
            ->append(" {$this->color}")
            ->append(" {$cardClasses}");
    }
```